### ISSUE TITLE
Feature request usability: Make clearer in which account I am. 

### ISSUE THREAD

**Issue by mxa** [TwidereProject/Twidere-Android#1143]
First off: thanks for this wonderful Twitter client.

I manage 5 accounts in Twidere.
I often re-tweet things with the wrong account because I lose track which account I am currently browsing. This can be bad because I am not supposed to have any political opinion with the company account for example.

Currently the only indication in which account I am is the _position_ of the colored bar beneath the top icons, which is in the one global color for the app.

Idea for improvement: Show this bar in the **color of the current account** and change the background color for the whole icon row to this color too.

**Comment by mariotaku** [comment]
What do you think if there's an account selector (like compose dialog does) in retweet dialog?

**Comment by mxa** [comment]
Yes, that could be another way of solving the issue. That would have the advantage of making the menu "open with account" unecessary and save a step there.

**Comment by trwnh** [comment]
It could also save having to long-press the icon
